package.json

```diff
@@ -29,6 +29,7 @@
     "framer-motion": "^10.3.0",
     "idb": "^7.1.1",
     "lodash.debounce": "^4.0.8",
+    "mixpanel-browser": "^2.47.0",
```

Comment:
So the custom transport wasn't needed in the end?

Reply:
Yes, looked closer at the docs and found a simpler/cleaner way